pkg/rc/replication_controller.go

```diff
@@ -35,9 +36,9 @@ type ReplicationController interface {
 	// The error channel will be closed in response.
 	WatchDesires(quit <-chan struct{}) <-chan error
 
-	// CurrentNodes() returns all nodes that this replication controller is currently scheduled on,
+	// CurrentPods() returns all nodes that this replication controller is currently scheduled on,
```

Comment:
not the result of this PR, but this comment is weird. the replication controller isn't scheduled on nodes